metals/src/main/scala/scala/meta/internal/metals/ConfiguredLanguageClient.scala

```diff
@@ -35,13 +35,13 @@ final class ConfiguredLanguageClient(
   }
 
   override def metalsStatus(params: MetalsStatusParams): Unit = {
-    if (config.statusBar.isOn) {
+    if (config.statusBar.isOn || clientCapabilities.statusBarIsOn) {
```

Comment:
I tried to order these in level of "support". So if either is fully turned on, we'll go with that, then showMessage, and then LogMessage.